common/scala/src/main/resources/logback.xml

```diff
@@ -0,0 +1,16 @@
+<configuration>
+  <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Recommend an AsyncAppender here; from the docs https://logback.qos.ch/manual/appenders.html
> This strategy has an amazingly favorable effect on performance at the cost of event loss.

(and its worth adding a comment to the config to advise on the lossy behavior when discarding TRACE/DEBUG/INFO events when writing to stream is backed up to 80% of queue size)

Reply:
Shouldn't give us any performance gain really, because our logging is already akka based and as such asynchronous from the code itself.